### ISSUE TITLE
PHOENIX dashboard

### ISSUE THREAD

**Issue by DanielAndreasen** [BrownDwarf/gollum#117]
This is related to #116.

* I think it needs to say where to get the example data. I found it here: https://github.com/OttoStruve/muler_example_data/tree/main/IGRINS/01_IGRINS_test_data
And mention all the three files that are needed for this
* Currently I get an error when creating the grid, since no points are found. I will try to debug, to make sure I didn't do anything wrong.

Review link: https://github.com/openjournals/joss-reviews/issues/6601

**Comment by DanielAndreasen** [comment]
For the grid, the structure of the PHOENIX models were slightly wrong (I will have to check if it is written correctly in the documentation a bit later, or if I did something wrong).
However, now the dashboard crashes due to some problems with the wavelength range. If I change the grid to: `grid = PHOENIXGrid(teff_range=(2500, 7000), logg_range=(2, 5), Z_range=(0, 0.5), wl_lo=21350, wl_hi=21590)` it works (I changed `wl_lo/wl_hi`).
This is not as smooth as the other dashboard, but I guess that is due to the longer wavelength range.

I think I nice addition to both examples is to write the actual parameters, Teff, logg, etc. so the user can "verify" their best fit. At least this is something I would like to have.

**Comment by Sujay-Shankar** [comment]
I'll take a look at all these issues and let you know when they've been fixed.
And yes, the dashboard does slow down as the wavelength range increases. A trick I've found to make it more responsive in such cases is to click through the sliders instead of dragging. This prevents the callbacks from stacking up and slowing the dashboard down.

**Comment by Sujay-Shankar** [comment]
Alright, I've changed the docs a little bit, they now give links to both the spec_a0v.fits and variance.fits files that IGRINSSpectrum needs to initialize (although gollum's repo now comes with them preloaded in the tutorial_data directory, which the docs also reflect now). The structure of the PHOENIX grid was a little inconsistent and confusing in the docs, I've updated them to be much easier to get up and running with little to no modification required after the download is complete. Basically all your config.env needs to point to is the directory which contains the wavelength file and the PHOENIX-ACES-AGSS-COND-2011 file (this specifically housing all the metallicity slices).

I've also added the verification for the best fit parameters (it is by eye, so not spot on obviously, but probably in the ballpark).
